sync-bucket.yml

```diff
@@ -0,0 +1,20 @@
+---
+platform: linux
+image_resource:
+  type: docker-image
+  source:
+    repository: 18fgsa/concourse-task
+
+inputs:
+  name: pipeline-tasks
+run:
+  path: pipeline-tasks/sync-bucket.sh
+
+params:
```

Comment:
Can you drop the variables in this file too? It will let us run `fly execute` against the task to test it as a job and be able to render the variables via `fly execute`.

Reply:
I don't understand this request - can you explain?

Reply:
I also disagree with this. Executing a single task outside a pipeline should be used to develop tasks, not as one-off ops functions.